src/main/java/com/minecolonies/coremod/client/gui/WindowHutBuilder.java

```diff
@@ -2,13 +2,24 @@
 
 import com.minecolonies.blockout.Pane;
 import com.minecolonies.blockout.controls.Label;
+import com.minecolonies.blockout.controls.Button;
+import com.minecolonies.blockout.Color;
 import com.minecolonies.blockout.views.ScrollingList;
 import com.minecolonies.blockout.views.SwitchView;
+import com.minecolonies.coremod.colony.buildings.AbstractBuilding;
 import com.minecolonies.coremod.colony.buildings.BuildingBuilder;
 import com.minecolonies.coremod.lib.Constants;
+import com.minecolonies.coremod.MineColonies;
+import com.minecolonies.coremod.network.messages.MarkBuildingDirtyMessage;
+import com.minecolonies.coremod.network.messages.TransferItemsRequestMessage;
+import com.minecolonies.coremod.util.InventoryUtils;
+import com.minecolonies.coremod.util.Log;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.coremod.util.Log'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)